site/src/pages/UsersPage/UsersPage.test.tsx

```diff
@@ -240,27 +240,35 @@ describe("UsersPage", () => {
 
   describe("pagination", () => {
     it("goes to next and previous page", async () => {
-      renderPage()
+      const { container } = renderPage()
```

Comment:
I love that we have tests and I'm happy to keep these in there. I'm curious if we should just add tests for the pagination widget itself, rather than tests for each time it's implemented.

Reply:
Yeah, I added a bit to the widget in a previous PR but I think there's also value in making sure the page is using the widget properly